examples/io/read_noise_covariance_matrix.py

```diff
@@ -14,13 +14,12 @@
 
 # %%
 
-from os import path as op
 import mne
 from mne.datasets import sample
 
 data_path = sample.data_path()
-fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis-cov.fif')
-fname_evo = op.join(data_path, 'MEG', 'sample', 'sample_audvis-ave.fif')
+fname_cov = data_path / 'MEG', 'sample', 'sample_audvis-cov.fif'
+fname_evo = data_path / 'MEG', 'sample', 'sample_audvis-ave.fif'
```

Comment:
this is not quite right.  All of the commas on these lines should change to `/`, not just the first comma.